framework/src/modules/validators/constants.ts

```diff
@@ -12,13 +12,9 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-import { BaseModule } from '..';
-import { ValidatorAPI } from './api';
-import { ValidatorEndpoint } from './endpoint';
-
-export class ValidatorModule extends BaseModule {
-	public id = 10;
-	public name = 'validator';
-	public api = new ValidatorAPI(this.id);
-	public endpoint = new ValidatorEndpoint(this.id);
-}
+export const MODULE_ID_VALIDATORS = 11; // TBD
+export const STORE_PREFIX_VALIDATORS_DATA = Buffer.from('0000');
```

Comment:
```suggestion
export const STORE_PREFIX_VALIDATORS_DATA = 0x0000;
```
it the implementation, we used `number` for prefix, so you could do like above. Otherwise, it needs to be converted again later